build.gradle

```diff
@@ -24,6 +24,13 @@ buildscript {
         classpath 'net.sf.proguard:proguard-gradle:6.1.1'
         classpath 'com.vanniktech:gradle-maven-publish-plugin:0.8.0'
     }
+
+    configurations.all {
+        resolutionStrategy {
+            // Use Proguard 6.2.0
+            force 'net.sf.proguard:proguard-gradle:6.2.0'
```

Comment:
Above Proguard is set to `6.1.1` in line 24. Just changing the version there would be fine without resolution strategy

Reply:
Good find! I tested it was using the correct version, but this was not intended, let me put up a fix.

Reply:
See: https://github.com/airbnb/MvRx/pull/310